ansible/roles/schulcloud-server-core/templates/admin-api-server-svc.yml.j2

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: admin-api-serv-svc
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: admin-api-serv
```

the labe shoud be **api-admin** 
```suggestion
    app: api-admin
```